src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
Link to the CSS file is incorrect. SCSS is a preprocessor and cannot be linked directly in HTML. You should compile the SCSS to CSS and then link the compiled CSS file instead.